app/src/main/java/org/mozilla/fenix/compose/list/ListItem.kt

```diff
@@ -48,6 +49,7 @@ private val ICON_SIZE = 24.dp
 @Composable
 fun TextListItem(
     label: String,
+    modifier: Modifier = Modifier,
```

Comment:
@MozillaNoah I will direct you to the `ListItem` changes

Reply:
Sweet. Do you want to add the modifiers to the other ListItems too while you're at it?

Reply:
I think this can be a followup.